engine/class_modules/sc_death_knight.cpp

```diff
@@ -8752,7 +8769,11 @@ void death_knight_t::create_buffs()
   // Conduits
   buffs.eradicating_blow = make_buff( this, "eradicating_blow", find_spell( 337936 ) )
         -> set_default_value( conduits.eradicating_blow.percent() )
-        -> set_trigger_spell( spec.obliterate );
+        -> set_trigger_spell( conduits.eradicating_blow );
+
+  buffs.unleashed_frenzy = make_buff( this, "unleashed_frenzy", find_spell( 338501 ) )
```

Comment:
Can also fetch the spellID from effectN(1).trigger() of the conduit data, doesn't change much but better for clarity